docs/options.md

```diff
@@ -191,12 +193,33 @@ CIBW_BEFORE_BUILD: pip install pybind11
 CIBW_BEFORE_BUILD: yum install -y libffi-dev && pip install .
 ```
 
-Platform-specific variants also available:  
+Platform-specific variants also available:<br/>
  `CIBW_BEFORE_BUILD_MACOS` | `CIBW_BEFORE_BUILD_WINDOWS` | `CIBW_BEFORE_BUILD_LINUX`
 
 
+### `CIBW_REPAIR_COMMAND` {: #repair-command}
+> Execute a shell command to repair each (non-pure Python) built wheel
+
+Default:
+- on Linux: `'auditwheel repair -w {dest_dir} {wheel}'`
+- on macOS: `'delocate-listdeps {wheel} && delocate-wheel -w {dest_dir} {wheel}'`
+- on Windows: `''`
+
+A shell command to repair a built wheel by copying external library dependencies into the wheel tree and relinking them.
+The command is run on each built wheel (except for pure Python ones) before testing it.
+
+The following placeholders must be used inside the command and will be replaced by `cibuildwheel`:
+- `{wheel}` for the absolute path to the built wheel
+- `{dest_dir}` for the absolute path of the directory where to create the repaired wheel.
+
+On Linux and macOS, the command is run in a shell, so you can write things like `cmd1 && cmd2`.
+
```

Comment:
~~~suggestion
#### Examples
```yaml
# don't repair mac wheels
CIBW_REPAIR_COMMAND_MACOS: "mv {wheel} {dest_dir}"

# pass the --lib-sdir flag to auditwheel on linux
CIBW_REPAIR_COMMAND_LINUX: "auditwheel repair --lib-sdir . -w {dest_dir} {wheel}"
```

~~~

Reply:
Should all the examples be moved after the "Platform-specific variants also available" bit?
It feels awkward to use the variants in examples before introducing them.

Reply:
Yes, I think you're right. Better to move all those "Platform-specific variants also available" bits before Examples. 

Would you mind making the change? Then we can merge.